estuary/app.py

```diff
@@ -109,6 +109,14 @@ def create_app(config_obj=None):
     app.register_error_handler(ServiceUnavailable, json_error)
     app.register_error_handler(AuthError, json_error)
     app.register_blueprint(api_v1, url_prefix='/api/v1')
+    try:
+        from estuary.api.monitoring import monitoring_api, configure_monitoring
+        app.register_blueprint(monitoring_api, url_prefix='/monitoring')
+        configure_monitoring(app)
+    except ImportError as e:
```

Comment:
Not sure how it would be visible for Estuary, but some log.warning saying that /metrics endpoint is disabled because of Prometheus client not installed would be nice here.

Reply:
Good idea. I added a warning.